docs/xostor/xostor.md

```diff
@@ -323,6 +323,78 @@ Only XCP-ng 8.2.1 is currently supported and with a maximum of 7 machines per po
 
 See this documentation: [RPU](/management/updates/#rolling-pool-update-rpu).
 
+## Upgrade
+
+If you are reading this documentation, we assume that you want to upgrade a pool on which XOSTOR is deployed, i.e. change the version of XCP-ng, for example from 8.2 to 8.3.
+For updates that don't change the version number of XCP-ng (bugfixes, security fixes), see [the update section](#update).
+
+### 1. Prerequisites
+
+- All hosts must be up to date on the version of XCP-ng you are currently using. For this refer to [the update section](#update).
+- HA must be disabled on your pool.
+- Ensure all nodes are reachable and resources are in "OK" state via XO's XOSTOR view. Alternatively, you can use the CLI:
+```
+linstor n l
+linstor r l
+linstor adv r
+linstor sp l
```

Comment:
Maybe not give the small command version? They can be prone to errors

Reply:
I agree. I think we should provide the full commands in the docs in general. That also better documents what the commands do, as the names have a meaning that is easier to understand than single letters.

Reply:
Don't see a problem here. Generally a simple copy-paste is executed by users.